### PR TITLE
Windows CI: Port TestExecApi* tests

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard jhoward@microsoft.com

Adds a few more tests to Windows to Windows CI - those in docker_api_exec_test.go. Results of local run below.

```
E:\Docker\build\busybox>runtest wl TestExecApi
Running test TestExecApi
DOCKER_TEST_HOST=tcp://127.0.0.1:2375
DOCKER_HOST=tcp://127.0.0.1:2375
=== RUN   Test
INFO: Testing against a local daemon
SKIP: docker_api_exec_test.go:50: DockerSuite.TestExecApiCreateContainerPaused (Test requires a Linux daemon)
PASS: docker_api_exec_test.go:18: DockerSuite.TestExecApiCreateNoCmd    5.903s
PASS: docker_api_exec_test.go:30: DockerSuite.TestExecApiCreateNoValidContentType       6.011s
SKIP: docker_api_exec_test.go:65: DockerSuite.TestExecApiStart (Test requires a Linux daemon)
PASS: docker_api_exec_test.go:88: DockerSuite.TestExecApiStartBackwardsCompatible       6.005s
PASS: docker_api_exec_test.go:102: DockerSuite.TestExecApiStartMultipleTimesError       6.625s
OK: 4 passed, 2 skipped
--- PASS: Test (33.22s)
PASS
ok      _/E_/go/src/github.com/docker/docker/integration-cli    33.507s
```

### CODE REVIEW

File: integration-cli/docker_api_exec_test.go [hunk 1]
```diff
@@ -100,12 +99,12 @@ func (s *DockerSuite) TestExecAPIStartBackwardsCompatible(c *check.C) {
...
 // #19362
-func (s *DockerSuite) TestExecAPIStartMultipleTimesError(c *check.C) {
-	dockerCmd(c, "run", "-d", "--name", "test", "busybox", "top")
+func (s *DockerSuite) TestExecApiStartMultipleTimesError(c *check.C) {
+	runSleepingContainer(c, "-d", "--name", "test")
 	execID := createExec(c, "test")
 	startExec(c, execID, http.StatusOK)
 
-	timeout := time.After(10 * time.Second)
+	timeout := time.After(60 * time.Second)
```
Comment (MHBauer) [inline integration-cli/docker_api_exec_test.go]: Is ten seconds cutting it too close on windows? Seemed fine in your example output.

Comment (jhowardmsft) [inline integration-cli/docker_api_exec_test.go]: I'm running these tests on a very fast workstation, not the Azure hosts ;) Generally I've used 60 as a blanket rule for all other tests I've ported. Better be safe than have a flakey test...

**Comment by calavera** [comment]
LGTM

File: integration-cli/docker_api_exec_test.go (hunk 1 above)
Comment (calavera) [inline integration-cli/docker_api_exec_test.go]: better to play safe :+1:

**Comment by vdemeester** [comment]
LGTM 🐮

**Pull request merged**